src/test/java/uk/gov/ida/matchingserviceadapter/validators/SubjectConfirmationValidatorTest.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.ida.matchingserviceadapter.validators;
+
+import org.beanplanet.messages.domain.Messages;
+import org.joda.time.Duration;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.opensaml.saml.saml2.core.SubjectConfirmation;
+import uk.gov.ida.saml.core.IdaSamlBootstrap;
+
+import static java.util.function.Function.identity;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.beanplanet.messages.domain.MessagesImpl.messages;
+import static uk.gov.ida.matchingserviceadapter.validators.SubjectConfirmationDataValidator.CONFIRMATION_DATA_NOT_PRESENT;
+import static uk.gov.ida.matchingserviceadapter.validators.SubjectConfirmationValidator.WRONG_SUBJECT_CONFIRMATION_METHOD;
+import static uk.gov.ida.saml.core.test.builders.SubjectConfirmationBuilder.aSubjectConfirmation;
+import static uk.gov.ida.saml.core.test.builders.SubjectConfirmationDataBuilder.aSubjectConfirmationData;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Please change to OpenSAMLMockitoRunner.class to ensure IdaSamlBootStrap.bootstrap() is called at test setup. This adds the OpenSaml marshalling types to the default un/marshaller.

Repeat for each test below also ...

Reply:
Thanks, didn't realise that existed!